pylab/website/templates/website/about.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% load trans blocktrans from i18n %}
+
+
+{% block content %}
+  <div class="about">
+    <h1>{% trans "About Summer Python Workshop" %}</h1>
+
+    <p>{% blocktrans %}During this event everybody can learn how to program in
+    Python and Django by creating <a href="http://pylab.lt">pylab.lt</a>
+    website.{% endblocktrans %}</p>
```

Comment:
Good point. Probably „Summer Python Workshop“ should not be mentioned here at all, because „Summer Python Workshop“ is just one of the events organized by „Python workshops“ community. And a note, who we start new projects, would good.

Also for now, I would suggest you to not translate this text, because it may change many times, so updating translations each time would be waste of time. First finish with the text, and then translate it.